tests/_ast/test_visitor.py

```diff
@@ -350,8 +370,9 @@ def foo(x=y, y=x, z=a):
     v.visit(mod)
     assert v.defs == set(["foo"])
     assert v.refs == set(["x", "y", "a"])
+    # TODO: Are these required refs?
```

Comment:
Yes, I think they are required refs -- can't define `foo` without them, right?

Reply:
Just checked, they are not: https://marimo.app/l/7swcsf

I'll take a look - but not sure it's worth trying to tease out this edgecase

Reply:
`del` just decrements the ref count though — `foo` still refers to the `Namespace` object, for example:

<img width="816" alt="image" src="https://github.com/marimo-team/marimo/assets/1994308/4b2e63b6-cbd0-4614-ae5c-7cee22d39e2b">

But I guess the symbols aren't required to be in globals, but instead are bound at function construction time.

Anyway I think it's fine as is